lib/active_resource/connection_ext.rb

```diff
@@ -5,6 +5,15 @@ class Connection
 
     attr_reader :response
 
+    alias_method :raw_initialize, :initialize
```

Comment:
Yeah. What do you think, should we merge this? A lot of people wanted it when it was discussed on Slack #developers.